src/python/deepgnn/migrate/data/ray_util.py

```diff
@@ -0,0 +1,116 @@
+"""Baseline Ray Trainer."""
```

Comment:
migrate folder is only for scripts to migrate to a new version. How `ray_util.py` is relevant to it?

Reply:
This is the block of code that gets added to user's folder, it contains a Ray trainer. This is copied to them instead of being a part of the library becase users will need to add metric tracking code and things like that directly.

Reply:
1. Put a common code that will work for 90% of use cases in the library.
2. Add an example to show how to use completely custom loop.